contracts/consensus-gateway/ConsensusCogateway.sol

```diff
@@ -94,6 +95,83 @@ contract ConsensusCogateway is MasterCopyNonUpgradable, MessageBus, ConsensusGat
         );
     }
 
+    /**
+     * @notice Confirm deposit in order to mint tokens.
+     *
+     * @param _amount MOST token deposit amount in atto.
+     * @param _beneficiary Address of beneficiary on auxiliary chain.
+     * @param _feeGasPrice Fee gas price at which rewards will be calculated.
+     * @param _feeGasLimit Fee gas limit at which rewards will be calculated.
+     * @param _depositor Address of depositor on the origin chain.
+     * @param _blockHeight Block height of origin chain at which storage proof
```

Comment:
blocks are now consistently referenced by a "number", `n`, and "height" is reserved for metablocks; just so that we can use "number" and "height" and know whether we're talking about blocks or metablocks (respectively)

```
```

Reply:
Make sense 👍